packages/siopv2-oid4vp-op-auth/src/session/functions.ts

```diff
@@ -130,6 +124,27 @@ export async function createOPBuilder({
   return builder
 }
 
+function getVerifyJwtCallback(
+  resolver?: Resolvable,
+  verifyOpts?: JWTVerifyOptions & {
+    checkLinkedDomain: 'never' | 'if_present' | 'always'
+    wellknownDIDVerifyCallback?: VerifyCallback
+  },
+): VerifyJwtCallback {
+  return async (jwtVerifier, jwt) => {
+    resolver = resolver ?? getResolver({ subjectSyntaxTypesSupported: ['ethr', 'ion'] })
+    const audience =
+      jwtVerifier.type === 'request-object'
+        ? (verifyOpts?.audience ?? getAudience(jwt.raw))
+        : jwtVerifier.type === 'id-token'
```

Comment:
This is a really odd ternary that could just be a single ternary with an or for the type

Reply:
simplified the ternary function.